crates/rolldown/src/ecmascript/format/esm.rs

```diff
@@ -294,15 +295,13 @@ fn create_import_declaration(
     }
     ret.push_str("{ ");
     ret.push_str(&specifiers.join(", "));
-    ret.push_str(" } from \"");
-    ret.push_str(path);
-    ret.push('"');
+    ret.push_str(" } from ");
+    ret.push_str(&escape(path));
```

Comment:
Wonder if we could reuse something from oxc. I remember there's a quiet a few symbols that needs to be handled sophisticately.

Reply:
Oxc's one is quite complex.

- https://github.com/oxc-project/oxc/blob/6787796cbafebb3e37aa96591b7b41797675ea6e/crates/oxc_codegen/src/str.rs#L28

I found two other implementation for the stringification.

- https://github.com/rolldown/rolldown/blob/88ef3e7fdf29ff6aa0686fd5594ed33f4eff9fc4/crates/rolldown_plugin_utils/src/to_string_literal.rs#L1
- https://github.com/rolldown/rolldown/blob/88ef3e7fdf29ff6aa0686fd5594ed33f4eff9fc4/crates/rolldown_plugin_utils/src/to_relative_runtime_path.rs#L81